examples/Patterns/Saga/InMemoryProvider.cs

```diff
@@ -65,5 +65,10 @@ public Task DeleteSnapshotsAsync(string actorName, long inclusiveToIndex)
         {
             throw new NotImplementedException();
         }
+
+        Task<long> IEventStore.PersistEventAsync(string actorName, long index, object @event)
```

Comment:
I meant this `IEventStore.PersistEventAsync` implementation should be deleted - the public Task PersistEventAsync on line 32 should be updated to return the correct value (Task.FromResult(nextEventIndex)) and the signature updated to match the interface (Task<long>), rather than us keeping two implementations of the same thing, otherwise it is confusing